ports/stm32/boards/Passport/modules/wallets/sw_wallets.py

```diff
@@ -20,6 +20,7 @@
 from .sparrow import SparrowWallet
 from .specter import SpecterWallet
 from .wasabi import WasabiWallet
+from .keeper import KeeperWallet
```

Comment:
Should be in alphabetic order 👀 . And the one below in `supported_software_wallets` too.